README.md

```diff
@@ -1,15 +1,15 @@
 # Github Actions for Convox
 
-This repository contains a set of Github Actions to be used with the [Convox](https://convox.com) Platform. If you [Install a Convox Rack](https://github.com/convox/installer) using the cloud provider of your choice and create a [convox.yml](https://docs.convox.com/application/convox-yml) to describe your application and its dependencies you can use these actions to create a complete CI/CD pipeline
+This repository contains links to a set of Github Actions to be used with the [Convox](https://convox.com) Platform. If you [Install a Convox Rack](https://github.com/convox/installer) using the cloud provider of your choice and create a [convox.yml](https://docs.convox.com/application/convox-yml) to describe your application and its dependencies you can use these actions to create a complete CI/CD pipeline
 
 ## Actions
-### [convox/actions/login](./login)
-Authenticates your Convox account You should run this action as the first step in your workflow
-### [convox/actions/deploy](./deploy)
+### [https://github.com/convox/action-login](.https://github.com/convox/action-login)
```

Comment:
Try out title "github.com/convox/action-login" and url "https://github.com/convox/action-login"